installation.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta http-equiv="Content-type" content="text/html;charset=UTF-8">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Code+Pro:300|Roboto+Mono:300,400">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" type="text/css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" type="text/css">
+  <link rel="stylesheet" href="./static/css/style.css" type="text/css">
+  <link rel="icon"  href="./static/img/favicon.ico" type="image/png">
+  <title>youtube-dl: Supported sites</title>
+  <meta http-equiv="Cache-Control" content="public, max-age=3600">
+</head>
+<body>
+
+<div class="wrapper supported">
+
+<header class="container-fluid">
+  <div class="container vertical-center">
+    <h1>Installation Manual</h1>
+  </div>
+</header>
+
+<div id="manual" class="container">
+
+  <p>Note that if you directly click on these hyperlinks, your web browser will most likely display the program contents. It's usually better to right-click on it and choose the appropriate option, normally called <em>Save Target As</em> or <em>Save Link As</em>, depending on the web browser you are using.</p>
```

Comment:
If any web browser indeed does display program contents, we may want to set better attributes and/or HTTP headers.
Can you name one or more browsers which do this? It sounds as though you assume it is not a bug in a single web browser, say IE6.